packages/block-library/src/post-featured-image/index.php

```diff
@@ -71,12 +80,19 @@ function render_block_core_post_featured_image( $attributes, $content, $block )
 		$featured_image = $featured_image . $overlay_markup;
 	}
 
-	$width  = ! empty( $attributes['width'] ) ? esc_attr( safecss_filter_attr( 'width:' . $attributes['width'] ) ) . ';' : '';
-	$height = ! empty( $attributes['height'] ) ? esc_attr( safecss_filter_attr( 'height:' . $attributes['height'] ) ) . ';' : '';
-	if ( ! $height && ! $width ) {
+	$aspect_ratio = ! empty( $attributes['aspectRatio'] )
+		? esc_attr( safecss_filter_attr( 'aspect-ratio:' . $attributes['aspectRatio'] ) ) . ';'
```

Comment:
is it possible that `safecss_filter_attr` doesn't allow `aspect-ratio`? it's removing it for me

Reply:
when I remove `safecss_filter_attr`, the frontend image shows the proper styles and looks like the editor

Reply:
you are right, I missed that! sorry